src/doc.d

```diff
@@ -345,7 +345,116 @@ extern (C++) static TemplateDeclaration getEponymousParent(Dsymbol s)
+extern (C++) __gshared const(char)* ddoc_default = "DDOC =  <html><head>
+        $(DDOC_COMMENT Generated by Ddoc from $(SRCFILENAME))
+        <META http-equiv=\"content-type\" content=\"text/html; charset=utf-8\">
+        <title>$(TITLE)</title>
+        </head><body>
+        <h1>$(TITLE)</h1>
+$(BODY)
+        <hr>$(SMALL Page generated by $(LINK2 http://dlang.org/ddoc.html, Ddoc). $(COPYRIGHT))
+        </body></html>
+
+B =     <b>$0</b>
+I =     <i>$0</i>
+U =     <u>$0</u>
+P =     <p>$0</p>
+DL =    <dl>$0</dl>
+DT =    <dt>$0</dt>
+DD =    <dd>$0</dd>
+TABLE = <table>$0</table>
+TR =    <tr>$0</tr>
+TH =    <th>$0</th>
+TD =    <td>$0</td>
+OL =    <ol>$0</ol>
+UL =    <ul>$0</ul>
+LI =    <li>$0</li>
+BIG =   <big>$0</big>
+SMALL = <small>$0</small>
+BR =    <br>
+LINK =  <a href=\"$0\">$0</a>
+LINK2 = <a href=\"$1\">$+</a>
+LPAREN= (
+RPAREN= )
+BACKTICK= `
+DOLLAR= $
+DEPRECATED= $0
+
```

Comment:
There's a subtle problem with empty lines in macro definitions - essentially the newline here is part of the DEPRECATED macro. To avoid that yet be able to insert logical separators, I've used this in .ddoc files:

```
_=-------------------------------------------
```

You may want to do the same.